src/components/Shifting/ShiftingCommentsSection.tsx

```diff
@@ -46,37 +44,50 @@ const CommentSection = (props: CommentSectionProps) => {
       pathParams={{ id: props.id }}
     >
       {(_, query) => (
-        <div className="flex w-full flex-col">
-          <textarea
-            rows={3}
-            value={commentBox}
-            minLength={3}
-            placeholder={t("type_your_comment")}
-            className="mt-4 rounded-lg border border-secondary-500 p-4 focus:ring-primary-500"
-            onChange={(e) => setCommentBox(e.target.value)}
-          />
-          <div className="flex w-full justify-end">
-            <ButtonV2
-              className="mt-4"
-              onClick={async () => {
-                await onSubmitComment();
-                query.refetch();
-              }}
-            >
-              {t("post_your_comment")}
-            </ButtonV2>
-          </div>
-          <div className="w-full">
+        <div className="flex w-full flex-col h-[700px] rounded-lg shadow-sm bg-white p-4 mt-2">
+          <div className="w-full flex flex-col grow overflow-y-auto">
             <PaginatedList.WhenLoading>
               <CircularProgress />
             </PaginatedList.WhenLoading>
+            <PaginatedList.WhenEmpty>
+              <div className="flex items-center justify-center text-gray-500 h-full my-24">
+                {t("no_comments_available")}
+              </div>
+            </PaginatedList.WhenEmpty>
+
             <PaginatedList.Items<CommentModel>>
               {(item) => <Comment {...item} />}
             </PaginatedList.Items>
             <div className="flex w-full items-center justify-center">
               <PaginatedList.Paginator hideIfSinglePage />
             </div>
           </div>
+
```

Comment:
<img width="503" alt="image" src="https://github.com/user-attachments/assets/0081be60-b390-49cd-8b5a-4a4943776fb2" />

Comments background need to be a bit darker.